fiftyone/operators/builtins/panels/model_evaluation/__init__.py

```diff
@@ -104,29 +105,32 @@ def get_avg_confidence(self, per_class_metrics):
                 total += metrics["confidence"]
         return total / count if count > 0 else None
 
-    def get_tp_fp_fn(self, ctx):
-        view_state = ctx.panel.get_state("view") or {}
-        key = view_state.get("key")
-        dataset = ctx.dataset
-        tp_key = f"{key}_tp"
-        fp_key = f"{key}_fp"
-        fn_key = f"{key}_fn"
-        tp_total = (
-            sum(ctx.dataset.values(tp_key))
-            if dataset.has_field(tp_key)
-            else None
-        )
-        fp_total = (
-            sum(ctx.dataset.values(fp_key))
-            if dataset.has_field(fp_key)
-            else None
-        )
-        fn_total = (
-            sum(ctx.dataset.values(fn_key))
-            if dataset.has_field(fn_key)
-            else None
-        )
-        return tp_total, fp_total, fn_total
+    def get_tp_fp_fn(self, info, results):
+        # Binary classification
```

Comment:
We show tp, fp, and fn only for object detection. Should we also show it for binary classification in the panel?

Reply:
It is valid to show TP/FP/FN for binary classification, so yes if you could show it in the UX that would be great!

Reply:
added a commit